invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/variables.less

```diff
@@ -12,6 +12,9 @@
 @linkHoverColor: darken(saturate(@linkColor, 20), 15, relative);
 @linkHoverUnderline: underline;
 
+@focusedFormBorderColor: #2185D0;
+@focusedFormMutedBorderColor: #2185D0;
```

Comment:
Does the `focusedFormMutedBorderColor` end up being necessary in the end?

Reply:
No, but I was thinking it makes sense to surface more theming variables rather than fewer, in general.  (Some of them reach into the depths of Semantic UI).  Curious to hear your thoughts on this issue.